ci/code_checks.sh

```diff
@@ -292,6 +292,10 @@ if mods:
     "
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    MSG='Check code for instances of os.remove' ; echo $MSG
+    invgrep -R --include="*.py*" --exclude "common.py" --exclude "test_writers.py" --exclude "test_store.py" -E "os.remove" pandas/tests/
```

Comment:
Do we need to escape the `.` in `os.remove`?

Reply:
I don't think so? `os.remove` is detected without escaping the `.` when files containing it are not excluded

Reply:
OK but like this I think it'll also detect `oszremove`. You'll need to escape the `.` else it'll match anything

Reply:
Sorry, got ya. Changed.

Reply:
cool, thanks - I'm not sure about the location of the check though, as it's currently inside the "checks on imported code" section. Should it be in "looking for unwanted patterns"?

Reply:
Makes sense, I've shifted it up to the 'patterns' section